### ISSUE TITLE
Ignore style sheets from another domain

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#228]
```
Third party libraries that already implement their own fixes for IE bugs 
break when used with ie7-js.  There should be a way to configure the library 
to ignore certain domains.  For example, ie7-css should have parameters 
(maybe an array of regular expressions) for matching domain names of 
stylesheets it should ignore.  Other parameters should configure certain IDs 
or classNames that should not be modified.

Example:
Setup the Meebo Bar on a website that uses ie7-js.  The Bar does not work as 
expected (incorrect DOM and styles).
```

Original issue reported on code.google.com by `mgh...@gmail.com` on 2 Mar 2010 at 1:59

**Comment by GoogleCodeExporter** [comment]
```
reminds me of http://code.google.com/p/ie7-js/issues/detail?id=44
```

Original comment by `fun...@gmail.com` on 2 Mar 2010 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
This is similar to issue #44 but not quite the same.

IE7.js cannot load style sheets from other domains anyway. So it should just 
ignore 
them instead of trying to parse them like it does currently.
```

Original comment by `dean.edw...@gmail.com` on 2 Mar 2010 at 9:51
- Changed title: **Ignore style sheets from another domain**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 3 Mar 2010 at 5:33
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Issue 242 has been merged into this issue.
```

Original comment by `dean.edw...@gmail.com` on 27 Apr 2010 at 7:44

**Comment by GoogleCodeExporter** [comment]
```
I'm changing this to "defect" as the script is currently throwing an error when 
it 
encounters a style sheet from a different domain.
```

Original comment by `dean.edw...@gmail.com` on 27 Apr 2010 at 7:45
- Added labels: **Priority-High**, **Type-Defect**
- Removed labels: **Priority-Medium**, **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 27 Apr 2010 at 8:25
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1 beta4.
```

Original comment by `dean.edw...@gmail.com` on 30 Apr 2010 at 9:14
- Changed state: **Fixed**
